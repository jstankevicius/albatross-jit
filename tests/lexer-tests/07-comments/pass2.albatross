everything in a comment is ignored # even the comment # symbol
# even bad names 0thisisabadname 1lakjsd
# even bad constants 0x_ _ 0x_473 0xCS473
