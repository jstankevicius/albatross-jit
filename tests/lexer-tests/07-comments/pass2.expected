1 1 NAME everything
12 1 NAME in
15 1 NAME a
17 1 NAME comment
25 1 NAME is
28 1 NAME ignored
